src/strategies/posichain-staking/index.ts

```diff
@@ -0,0 +1,38 @@
+import { StaticJsonRpcProvider } from '@ethersproject/providers';
+import { formatUnits } from '@ethersproject/units';
+import { BigNumber } from '@ethersproject/bignumber';
+
+export const author = 'dannyposi';
+export const version = '0.0.1';
+
+type Params = {
+  symbol: string;
+  decimals: number;
+};
+
+export async function strategy(
+  _space: string,
+  _network: string,
+  provider: StaticJsonRpcProvider,
+  _addresses: Array<string>,
```

Comment:
You were not using _addresses? The response doesn't depend on addresses.

Reply:
I've add comment to explain about that. I see strategy caller has filtered out all addresses that not needed: https://github.com/snapshot-labs/snapshot-strategies/blob/02439eb120ed7c4cc0c493924b78d92d22006b40/src/utils.ts#L26